### ISSUE TITLE
unused dependencies showing 'object'

### ISSUE THREAD

**Issue by helio-frota** [bucharest-gold/szero#61]
```
----------------------------------------------------------------------
[ Unused dependencies ]
----------------------------------------------------------------------
[object Object],[object Object]
```

**Comment by helio-frota** [comment]
tested with this PR:
https://github.com/hawkular/hawkular-apm-opentracing-javascript/pull/11

**Comment by lholmquist** [comment]
looks like this is **not** happening in version 0.6.0

**Comment by helio-frota** [comment]
sounds like an introduced bug ?

**Comment by lholmquist** [comment]
@helio-frota yup.  i'm on the case!!

**Comment by lholmquist** [comment]
landed in 987e782
